### ISSUE TITLE
nvda does not announce the names of the menus when walking with left and right arrows in the itunes 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#12331]
### Steps to reproduce:
open itunes.
press alt
walk with arrows to the right and left 
### Actual behavior:
nvda goes mute 
### Expected behavior:
nvda should read the names of the menus in focus 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-22498,d9afe35d
#### Windows version:
10 10.0.19043,962
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:
[itunes.txt](https://github.com/nvaccess/nvda/files/6367990/itunes.txt)

**Comment by Adriani90** [comment]
Note that iTunes for Windows will probably be discontinued at some point in time in favor of other products such as apple music and apple tv. For Mac, iTunes has been discontinued already in 2019.
Apple provides so far only maintenance updates to iTunes for Windows.
https://www.businessinsider.com/itunes

The Windows version is expected to stay alive for some more time.
https://www.groovypost.com/unplugged/apple-itunes-might-be-on-life-support-but-it-isnt-dead/

This particular issue occurs in other screen readers like Jaws as well, so it needs to be fixed by Apple. However, I don't expect further accessibility updates to iTunes unless screen reader users try to push it at Apple.

**Comment by Adriani90** [comment]
Given this article and that alternatives from Apple for iTunes are already published in the Windows store, I am closing as abandoned.
https://applescoop.org/story/from-icon-to-obsolete-what-happened-to-apple-itunes

If you want to hear your Apple podcasts on your Windows PC, there is also a new alternative from Apple:
https://www.groovypost.com/howto/listen-apple-podcasts-pc-web-app/

For audiobooks there is no solution yet, but it is a matter of  short time until it will come. If you have audiobooks on iTunes and want to export them, there are plenty of converters that can convert them to mp3.
